client/src/components/home/home.styles.scss

```diff
@@ -1,63 +1,135 @@
 .home-container{
-    padding-top:10%;
-}
-.button-container {
-    display:flex;
-    justify-content: center;
-    align-items: center;
-    flex-direction: column;
+    padding-top: 10%;
+    .main-container{
+        display:flex;
+        justify-content: center;
+        flex-direction: column;
+        padding: 3% 2% 3%;
+        margin-top: 1%;
+        margin-bottom: 0.5%;
+        border-bottom: 1px solid rgb(232, 232, 232);
+        border-top: 1px solid rgb(232, 232, 232);
+        border-left: 0.2px solid rgb(232, 232, 232);
+        border-right: 0.2px solid rgb(232, 232, 232);
+        background: rgb(255,255,255);
+        border-bottom-left-radius: 20px;
+        border-bottom-right-radius: 20px;
+        border-top-left-radius: 20px;
+        border-top-right-radius: 20px;
+        background-color: #fafbff;
+        width:70%;
+        margin: auto;
+        .button-container {
+            display:flex;
+            justify-content: center;
+            align-items: center;
+            flex-direction: column;
+            margin-top: 20px;
+        }
+        
+        .button {
+            font-family:'Yanone Kaffeesatz' ,  'sans-serif';
+            color:#464646;
+            padding: 8px 10%;
+            line-height: 1.75;
+            background-color: white;
+            border: 0;
+            cursor: pointer;
+            margin: 10px;
+            display: inline-flex;
+            justify-content: center;
+            outline: 0;
+            border-radius: 25px;
+            box-shadow: -0.5px -0.5px 0.5px 0.5px rgba(0, 0, 0, 0.19), 0.5px 0.5px 0.5px 0.5px rgba(0, 0, 0, 0.19);
+            background-color: #fcfcfd;
+            width:25%;
+            &:hover {
+                background-color: #007dd1;
+                transition:0.7s;
+            }
+        }
+        
+        .logo-container{
+            display: flex;
+            flex-direction: row;
+            justify-content: center;
+            align-self: center;
+            padding: 1% 2% 1%;
+            margin-top: 1%;
+            margin-bottom: 0.5%;
+            border-bottom: 1px solid rgb(232, 232, 232);
+            border-top: 1px solid rgb(232, 232, 232);
+            border-left: 0.2px solid rgb(232, 232, 232);
+            border-right: 0.2px solid rgb(232, 232, 232);
+            margin: auto;
+            width: 70%;
```

Comment:
Can we write structural css (like width, height, padding, margin) in the beginning of the css class due to better readibility